externals/kyuubi-jdbc-engine/src/test/resources/impala-compose.yml

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# modified compose file from Impala repo, see
+# https://github.com/apache/impala/blob/4.3.0/docker/quickstart.yml
+version: "3.5"
+services:
+  metastore:
+    image: apache/impala:4.0.0-impala_quickstart_hms
```

Comment:
possible to use the latest 4.3? or are there good reasons to stick to 4.0?

Reply:
There are no official impala quickstart images for version 4.3 in [Dockerhub](https://hub.docker.com/r/apache/impala/tags), images from compose file are the latest ones. I also tried to use other user-built images, but they were either too old or too large.